python/ray/util/sgd/torch/training_operator.py

```diff
@@ -237,15 +237,18 @@ def train_batch(self, batch, batch_info):
                 calculate averages.
 
         """
-        features, target = batch
+        # unpack features into list to support multiple inputs model
+        *features, target = batch
```

Comment:
Looks good. can you update the docstring?

Reply:
updated